go.mod

```diff
@@ -1,9 +1,11 @@
 module go.signoz.io/signoz
 
-go 1.21.3
+go 1.22.0
+
+toolchain go1.22.7
```

Comment:
this is required for the changes present in this PR ?

Reply:
Yes